you-dont-need/You-Dont-Need-JavaScript
# [Fix]: Rendering of appropriate social icons on webpage, [Feature]: Customized footer to more aesthetic UI

## Customization of current footer:
Fixes #352 

## Issues resolved:
* Successful rendering of social media on webpage
*  Custom UI to footer for more appealing and aesthetic look
*  Hover effects and matching color scheme

## How to check:
* Navigate to "Responsive footer" folder
* Open index.html locally using liver sever
* You can clearly observe the changes

## Before:
![Screenshot 2023-10-23 222015](https://github.com/you-dont-need/You-Dont-Need-JavaScript/assets/119743101/35612b46-b81b-4d5f-b63d-ea5aff6a89d8)

## After:
![Screenshot 2023-10-23 231629](https://github.com/you-dont-need/You-Dont-Need-JavaScript/assets/119743101/5d872964-9616-4a5d-bdf0-cf395c6c33e0)
![Screenshot 2023-10-23 231646](https://github.com/you-dont-need/You-Dont-Need-JavaScript/assets/119743101/d78c3339-d962-4552-b636-06ee762f9941)
